src/Input/Silk.NET.Input.Common/Interfaces/IKeyboard.cs

```diff
@@ -15,6 +15,11 @@ public interface IKeyboard : IInputDevice
         /// The keys this keyboard has available.
         /// </summary>
         IReadOnlyList<Key> SupportedKeys { get; }
+        
+        /// <summary>
+        /// The text currently in the clipboard
```

Comment:
XML Comments should end on a period:

```suggestion
        /// The text currently in the clipboard.
```